go-selinux/label/label_selinux.go

```diff
@@ -101,9 +101,17 @@ func FormatMountLabel(src, mountLabel string) string {
 // SetProcessLabel takes a process label and tells the kernel to assign the
 // label to the next program executed by the current process.
 func SetProcessLabel(processLabel string) error {
+	if processLabel == "" && selinux.GetEnabled() {
+		processLabel = "unconfined_u:unconfined_r:unconfined_t:s0"
```

Comment:
@rhatdan Does it make sense?

Reply:
Or after failure with `""`, then we try `"unconfined_u:unconfined_r:unconfined_t:s0"`?

Reply:
No we should not hard code this label.  "" is a valid label.

Reply:
It tells the the library to reset the kernel to use default labeling.